docs-new/docs/use-cases/non-k8s.md

```diff
@@ -1,29 +1,44 @@
-# Keptn - Kubernetes
+# Keptn for non-Kubernetes deployments
 
-Keptn Tasks running on a Kubernetes cluster
-can be triggered for [workloads](https://kubernetes.io/docs/concepts/workloads/) and applications
-that are deployed outside of Kubernetes.
-For example, Keptn could trigger load and performance tests
-for an application that is deployed on a virtual machine.
-
-To do this:
+Keptn can interact with deployments that are not running on Kubernetes
+even though Keptn mainly targets Kubernetes deployments.
+The following functionality is available for deployments not on Kubernetes:
 
-- [Install Keptn on a Kubernetes cluster](#install-keptn-on-a-kubernetes-cluster)
-- [Create a KeptnTaskDefinition](#create-a-keptntaskdefinition)
-- [Create and apply a KeptnTask](#create-and-apply-a-keptntask)
-- [Re-run the KeptnTask](#re-run-the-keptntask)
+- [Run Keptn tasks](#run-keptntask-for-a-deployment-not-on-kubernetes)
+- [Run Keptn analysis](#run-keptn-analysis-for-a-deployment-not-on-kubernetes)
 
-## Install Keptn on a Kubernetes cluster
-
-You must set up a Kubernetes cluster and
+To use either of these features,
+you must set up a Kubernetes cluster and
 [install](../installation/index.md#basic-installation)
 Keptn on it,
 but this can be a very lightweight, single-node KinD cluster; see
 [Create local Kubernetes cluster](../installation/k8s.md#create-local-kubernetes-cluster).
-Keptn only triggers on-demand `KeptnTask` resources
+Keptn only runs  on-demand `KeptnTask` and `KeptnAnalysis` resources
 so resource utilization is minimal.
 
-## Create a KeptnTaskDefinition
+## Run KeptnTask for a deployment not on Kubernetes
+
+Keptn tasks running on a Kubernetes cluster can be triggered for
+[workloads](https://kubernetes.io/docs/concepts/workloads/)
+and applications that are deployed outside of Kubernetes.
+For example, Keptn can run (or "trigger")
+load and performance tests
+for an application that is deployed on a virtual machine.
```

Comment:
```suggestion
for an application that is deployed on a virtual machine,
or any other environment, as long as it can be reached
by the container running the KeptnTask.
```